tests/test_nan_inf.py

```diff
@@ -0,0 +1,38 @@
+"""Test serialization of nan and inf values."""
+import math
+
+import pytest
+
+import helpers
+
+program_code = """
+    parameters {
+      real eta;
+      real positive_infinity;
```

Comment:
why are these defined in parameters block? Aren't these special functions?

Reply:
I don't know. It seems like this is how it is done in the Stan language. See https://github.com/stan-dev/stan/blob/develop/src/test/test-models/good/nullary-unconflicted.stan

If there's a better way, I'm happy to change things. This seems to work.

Reply:
I was just reading from here https://mc-stan.org/docs/2_25/functions-reference/special-values.html

Reply:
I'm really short on time and I'd like to get httpstan and pystan fixed. If you think there's a better way to write this test---and there certainly may be a better way---perhaps we could discuss it in a new issue?